core/src/main/java/nl/nn/adapterframework/webcontrol/api/ExecuteJdbcQuery.java

```diff
@@ -130,30 +129,23 @@ public Response execute(LinkedHashMap<String, Object> json) throws ApiException
 			qs.setBlobSmartGet(true);
 			qs.configure(true);
 			qs.open();
-			result = qs.sendMessage(new Message(query), null).asString();
+			Message message = qs.sendMessage(new Message(query), null);
+			result = message.asString();
```

Comment:
Could you turn the resultType in an if else case and if none matches do asString?